packages/oscal-react-library/src/components/OSCALCatalogGroups.js

```diff
@@ -112,9 +119,12 @@ export default function OSCALCatalogGroups(props) {
     }
     // Confirm catalog tab group can be grabbed
     if (document.getElementById(`vertical-tab-${rootLayer}`)) {
+      console.log("Heyo");
```

Comment:
```suggestion
```

Reply:
You're telling me we don't need a "Heyo" logged every time a Catalog group tab is matched to part of a fragment? 😂